Sandboxes

jezhumble opened the pull request:
I've added a new page for sandboxes, and revised the Infrastructure and AWS pages (the latter substantially) to reflect how I think AWS (both prod and sandboxes) should work in future. Please review!

afeld commented on _pages/infrastructure/aws.md:
```diff
@@ -3,53 +3,52 @@ title: Amazon Web Services
+### OS baseline
 
-Every inter-agency agreement (IAA) at 18F needs to have a line item on a total value to spend on infrastructure, including Amazon Web Services (AWS). Unless it is part of a negotiation with 18F Infrastructure, we *do not* pay for non-production hosting costs. All costs must go back to the Federal partner or another funding source. There is no actual concept of _non-billable_ - there are only cost ...
+We use a pre-hardened version of [Ubuntu](https://en.wikipedia.org/wiki/Ubuntu_%28operating_system%29) as our baseline OS for all EC2 instances in AWS. In AWS, there are [Amazon Machine Images](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) (AMIs) in each AWS Region that these controls already implemented. You should always launch new instances from this baseline.
```
Can we add information (or a link) about how to find it?

afeld replied:
Also, it seems that this paragraph now exists on two pages.

jezhumble commented:
@afeld - thanks very much for your feedback. I've updated the text: please do let me know if it addresses your concerns or if there's other stuff that needs more clarity.

NoahKunin commented:
Looks great - merging!

afeld commented:
Thanks for this, gents!
